openpype/settings/defaults/project_settings/global.json

```diff
@@ -271,28 +271,7 @@
             }
         }
     },
-    "project_folder_structure": {
-        "__project_root__": {
-            "prod": {},
-            "resources": {
-                "footage": {
-                    "plates": {},
-                    "offline": {}
-                },
-                "audio": {},
-                "art_dept": {}
-            },
-            "editorial": {},
-            "assets[ftrack.Library]": {
-                "characters[ftrack]": {},
-                "locations[ftrack]": {}
-            },
-            "shots[ftrack.Sequence]": {
-                "scripts": {},
-                "editorial[ftrack.Folder]": {}
-            }
-        }
-    },
+    "project_folder_structure": "{\"__project_root__\": {\"prod\": {}, \"resources\": {\"footage\": {\"plates\": {}, \"offline\": {}}, \"audio\": {}, \"art_dept\": {}}, \"editorial\": {}, \"assets[ftrack.Library]\": {\"characters[ftrack]\": {}, \"locations[ftrack]\": {}}, \"shots[ftrack.Sequence]\": {\"scripts\": {}, \"editorial[ftrack.Folder]\": {}}}}",
```

Comment:
(this is just a side note) 
This is still not quite ok to use Ftrack objects in openpype global settings. It seems to be there has to be a better way to add Ftrack related settings. Ftrack module settings could be used (rather than here) - so we keep openpype independent for those facilities which are not using track at all.

Reply:
Yes agree but that is not related to this PR.

Reply:
In that case - is there any related issue we could link to this?

Reply:
Created https://github.com/pypeclub/OpenPype/issues/1816